trillian/integration/hammer.go

```diff
@@ -141,6 +141,10 @@ type HammerConfig struct {
 	MaxRetryDuration time.Duration
 	// RequestDeadline indicates the deadline to set on each request to the log.
 	RequestDeadline time.Duration
+	// DuplicateChance sets the probability of attempting to add a duplicate when
+	// calling add[-pre]-chain (as the N in 1-in-N). Set to 0 to disable sending
+	// duplicates.
+	DuplicateChance int
```

Comment:
It's a slight shame this has to be different to the existing `Bias` mechanisms but they seem to be handler specific. 

Not sure if DuplicateFraction would be easier to understand and then have 0-1 random values.

Reply:
The 1-in-N thing already exists in the codebase so it seemed better to stick with that.

Reply:
OK.